### PR TITLE
[ADD] l10n_es: add cooperatives account chart template

### PR DESCRIPTION
**Pull request by acysos**
Hello,

This PR add the PCGE for Spanish Cooperatives pymes and complete.
This PGCE make some change in groups because add new groups only for this type of companies. So it have a common account chart template with the groups that are common and two new templates for each cooperative.

It neccesary to change some accounts in the common account chart template because have different name. This is the list of account removed por account chart template common and added in each chart:

- 112
- 113
- 150
- 171
- 502
- 521
- 526
- 5585

More information about Spanish Cooperative Chart Account here: https://www.boe.es/buscar/doc.php?id=BOE-A-2010-20034

Greetings

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by acysos** [comment]
Hello, 

I see that I remove the doble colon in the CSV when I edited. I go to send again.

Greetings

**Comment by acysos** [comment]
Hello,

I have change the CSV with the double quotes. The PR can continue.

Greetings

**Comment by ValentinVinagre** [comment]
@HaraldPanten

File: addons/l10n_es/data/account_chart_template_data.xml [hunk 1]
```diff
@@ -39,5 +39,33 @@
         <field name="transfer_account_code_prefix">572999</field>
         <field name="parent_id" ref="account_chart_template_common"/>
     </record>
+    <record id="account_chart_template_coop_common" model="account.chart.template">
+        <field name="name">PGCE Cooperativas común</field>
```
Comment (HaraldPanten) [inline addons/l10n_es/data/account_chart_template_data.xml:43]: If we want to continue with the same structure as the other COA, "Común" should be written with capital letter. Shouldn't it?

File: addons/l10n_es/data/account_group.xml [hunk 2]
```diff
@@ -281,11 +301,21 @@
...
     <record id="account_group_153" model="account.group.template">
         <field name="chart_template_id" ref="account_chart_template_common"/>
         <field name="code_prefix_start">153</field>
         <field name="name">Desembolsos no exigidos por acciones o participaciones consideradas como pasivos financieros</field>
     </record>
+    <record id="account_group_1530" model="account.group.template">
+        <field name="chart_template_id" ref="account_chart_template_coop_common"/>
+        <field name="code_prefix_start">1530</field>
+        <field name="name">Desembolso no exigos por Capital social cooperativo considerado como pasivo financiero</field>
+    </record>
```
Comment (HaraldPanten) [inline addons/l10n_es/data/account_group.xml:318]: Desembolsos no exigidos

File: addons/l10n_es/data/account_group.xml [hunk 3]
```diff
@@ -2126,6 +2166,11 @@
         <field name="code_prefix_start">5297</field>
         <field name="name">Provisión a corto plazo por transacciones con pagos basados en instrumentos de patrimonio</field>
     </record>
+    <record id="account_group_5298" model="account.group.template">
+        <field name="chart_template_id" ref="account_chart_template_coop_common"/>
+        <field name="code_prefix_start">5298</field>
+        <field name="name">Fondo de educacion, formacion y promocion a corto plazo</field>
+    </record>
```
Comment (HaraldPanten) [inline addons/l10n_es/data/account_group.xml:2173]: You are writing "Fondo de Educación, Formación y Promoción a largo plazo" for long term. I think you should use capital letters here, and solve typos.

File: addons/l10n_es/data/account_group.xml [hunk 4]
```diff
@@ -2731,6 +2776,11 @@
         <field name="code_prefix_start">602</field>
         <field name="name">Compras de otros aprovisionamientos</field>
     </record>
+    <record id="account_group_605" model="account.group.template">
+        <field name="chart_template_id" ref="account_chart_template_coop_common"/>
+        <field name="code_prefix_start">605</field>
+        <field name="name">Compras efectuados a socios</field>
+    </record>
```
Comment (HaraldPanten) [inline addons/l10n_es/data/account_group.xml:2783]: Efectuadas

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Just some writing changes

**Comment by acysos** [comment]
@HaraldPanten hecho

**Comment by HaraldPanten** [comment]
Hi @acysos 

I still see accent marks that you should add in Educacion, Formacion y Promocion --> Educación, Formación y Promoción

Could you check the commit message as well? I see 2 [ADD] messages that are meaning the same.

Sorry for being picky 😝.

THX!

**Comment by acysos** [comment]
> Hi @acysos
> 
> I still see accent marks that you should add in Educacion, Formacion y Promocion --> Educación, Formación y Promoción
> 
> Could you check the commit message as well? I see 2 [ADD] messages that are meaning the same.
> 
> Sorry for being picky stuck_out_tongue_closed_eyes.
> 
> THX!

xD Done

**Review by HaraldPanten — APPROVED** [review]
We could add this one and make changes in a future if needed.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Pull request closed without merging**